lib/client.js

```diff
@@ -107,10 +111,19 @@ Client.prototype._attachApi = function () {
       self._connection.host
     );
 
-    self._swagger = new swagger.SwaggerApi({
-      url: ariUrl,
-      success: swaggerLoaded,
-      failure: swaggerFailed
+    request(ariUrl, function (err) {
+        if (err && hostIsNotReachableErrorCodes.indexOf(err.code) !== -1) {
+            err.name = 'HostIsNotReachable';
+
+            self.emit('APILoadError', err);
+            reject(err);
+      } else {
```

Comment:
indentation here needs to be fixed. It's mixing 2 and 4 spaces looks like.

Reply:
Done